### PR TITLE
Introduce run_linter.sh to lint Python files.

### PR DESCRIPTION
**Pull request by hoangviet1993**
For Python files, I can see that the main Oppia app is using
- [x] Pylint with config file .pylintrc
- [x] Pydocstyle
- [x] Pycodestyle with config file tox.ini

Will probably get to JS/HTML/CSS linter at some point.

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -9,27 +9,49 @@ init-hook='import sys; sys.path.append("../oppia_tools/google_appengine_1.9.67/g
...
+class-naming-style=PascalCase
 
 # Regex for dummy variables (to prevent 'unused argument' errors)
 dummy-variables-rgx=_|unused_*
 
+# Regular expression which should only match correct function names
+function-rgx=^[_a-z][a-z0-9_]*$
+
+# Good variable names which should always be accepted, separated by a comma
+good-names=e,_,d,f,i,l,p,w,fn,fs,id,pc,sc,zf,setUp,tearDown,longMessage,maxDiff
```
Comment (seanlip) [inline .pylintrc]: Not sure you need most of this FYI.

File: app.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ threadsafe: true
 
 handlers:
 - url: /ajax/.*
-  script: main.app
+  script: main.APP
```
Comment (seanlip) [inline app.yaml]: Why this change? I think lowercase is fine...

File: core/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+"""Core files for backend functionalities."""
```
Comment (seanlip) [inline core/__init__.py]: This doesn't need a docstring, it's fine (and probably better) for `__init__` files to be blank.

File: app.yaml (hunk 2 above)
Comment (hoangviet1993) [reply app.yaml]: I changed the main.py `app` class to `APP` to comply with Pylint constant variable rule (all uppercase) so it has to be updated here in the app.yaml.

File: core/__init__.py (hunk 3 above)
Comment (hoangviet1993) [reply core/__init__.py]: This is from pydocstyle(PEP8)
https://media.readthedocs.org/pdf/pydocstyle/2.1.1/pydocstyle.pdf

D107 Missing docstring in __init__

Am I making the right call in trying to comply with both Pylint and PEP8?

Comment (seanlip) [reply core/__init__.py]: No, you don't need to follow all the pylint rules. We can make exclusions as needed. I think this is one of them, and the APP thing is another.

File: .pylintrc (hunk 1 above)
Comment (hoangviet1993) [reply .pylintrc]: Got rid of most of them except `setUp` and `tearDown`!

File: core/__init__.py (hunk 3 above)
Comment (hoangviet1993) [reply core/__init__.py]: Gotcha! Done :)

File: app.yaml (hunk 2 above)
Comment (hoangviet1993) [reply app.yaml]: Aite, I put things back to where it was and added override Pylint rule.

**Comment by hoangviet1993** [comment]
@seanlip PTAL

**Comment by seanlip** [comment]
Thanks!
